tests/build.gradle

```diff
@@ -48,11 +48,9 @@ dependencies {
     compile 'junit:junit:4.11'
     compile 'com.jayway.restassured:rest-assured:2.6.0'
     compile 'org.scalatest:scalatest_2.11:3.0.1'
-    compile 'io.spray:spray-testkit_2.11:1.3.3'
-    compile 'com.typesafe.akka:akka-testkit_2.11:2.4.16'
```

Comment:
Tests failed when I added akka-testkit. It looks like the tests are using `ScalatestRouteTest`, which was moved from akka-testkit to akka-http-testkit, so the former isn't needed in the build.gradle file.